README.md

```diff
@@ -1,2 +1,112 @@
-# aue
-Calculations for best- and worst-case scenario adult-use establishment distributions
+# Adult Use Establishments Proximity Analysis
+
+This repo contains code for identifying the best- and worst-case scenarios for the spatial distribution of Adult Use Establishments in NYC, given the restriction that no establishment be within 500ft of another establishment.
```

Comment:
"of Adult Use Establishments in NYC on a subset of lots where Adult Use Establishments are allowed"
or something along those lines.  After reading this though once I think it might be useful to mention that we're working with a subset of PLUTO lots